src/managers/commands.js

```diff
@@ -76,6 +77,13 @@ class Commands {
         }
     }
 
+    handleReactions(msg) {
+        if (msg.channel.id === this.bot.config.introductionsChannel) {
+            const [ctSemicolon] = emojis.ct_semicolon.match(/\d+/);
```

Comment:
One small change I'd like to request for the sake of consistency, you don't have to get the id from the emoji itself, you are able to react with the full string that is present in the emoji.js file as I did [here](https://github.com/CodingTrain/trainbot/blob/master/src/commands/information/userinfo.js#L12-L15).